source/Subsystem/Renderer/Console.cpp

```diff
@@ -0,0 +1,342 @@
+#include "Console.h"
```

Comment:
I guess this file was made through a tutorial? We might want to optimize making new windows later 👍

Reply:
The original was part of imgui itself, I just modified it a bunch to make it work in our application.